src/Raven.Client/Documents/Operations/Backups/RestoreBackupConfiguration.cs

```diff
@@ -40,6 +40,8 @@ public class RestoreBackupConfiguration : RestoreBackupConfigurationBase
     {
         public string BackupLocation { get; set; }
 
+        public string NodeTag { get; set; }
```

Comment:
Why do you need the `NodeTag` here? (this isn't used in the restore process)
You can add another parameter to `RestoreDatabase`.

Reply:
I want to restore it to a specific node.

Reply:
It's already handled at the operation level

Reply:
Removed this property.